Source/DataSources/EntityView.js

```diff
@@ -73,7 +82,11 @@ define([
                     var inertialCartesian = Matrix3.multiplyByVector(toInertial, cartesian, updateTransformCartesian3Scratch5);
                     var inertialDeltaCartesian = Matrix3.multiplyByVector(toInertialDelta, deltaCartesian, updateTransformCartesian3Scratch6);
 
-                    Cartesian3.subtract(inertialCartesian, inertialDeltaCartesian, updateTransformCartesian3Scratch4);
+                    if (invertVelocity) {
+                        Cartesian3.subtract(inertialDeltaCartesian, inertialCartesian, updateTransformCartesian3Scratch4);
```

Comment:
Looking at this more closely, the result here gets fed straight into `Cartesian3.magnitude` and not used again.  So I don't think the `invertVelocity` flag needs to be checked here at all, as magnitude yields an absolute value.  The flag does need to be checked in the VVLH section below for `yBasis`.